regolith/validators.py

```diff
@@ -109,4 +113,35 @@ def ensure_email(email):
     'databases': (always_false, ensure_databases),
     'stores': (always_false, ensure_stores),
     'email': (always_false, ensure_email),
-    }
+}
+
+
+def pop_description(schema):
+    """Pop description off the schema since Cerberus doesn't support it"""
+    if isinstance(schema, dict):
+        schema.pop('description', None)
+        for v in schema.values():
+            pop_description(v)
+
+
+def validate(db, record):
+    """Validate a record for a given db
+    
+    Parameters
+    ----------
+    db : str
+        The name of the db in question
+    record : dict
+        The record to be validated
+
+    Returns
+    -------
+    bool:
+        True is valid
+
+    """
+    from regolith.schemas import SCHEMAS
```

Comment:
There is no circular importing here. Please move this import to the top of the file.

Reply:
Or maybe, better yet, move this function to schemas.py or SCHEMAs to this module.  Modules with only one object don't deserve to exist :)